src/main/java/uk/gov/justice/probation/courtcasematcher/service/MatcherService.java

```diff
@@ -80,12 +77,46 @@
                         .build());
     }
 
+    // TODO - when offender search gives us the probation status we can remove this method
+    @Deprecated(forRemoval = true)
+    private Mono<ProbationStatusDetail> getProbationStatus(Offender offender) {
+        if (featureFlags.offenderSearchForProbationStatus()) {
+            var probationStatus = offender.getProbationStatus();
+            return Mono.just(ProbationStatusDetail.builder()
+                .probationStatus(Optional.ofNullable(probationStatus).map(ProbationStatus::getStatus).orElse(null))
+                .inBreach(Optional.ofNullable(probationStatus).map(ProbationStatus::getInBreach).orElse(null))
+                .previouslyKnownTerminationDate(Optional.ofNullable(probationStatus).map(ProbationStatus::getPreviouslyKnownTerminationDate).orElse(null))
+                .preSentenceActivity(Optional.ofNullable(probationStatus).map(ProbationStatus::isPreSentenceActivity).orElse(false))
+                .build());
+        }
+        return restClient.getProbationStatus(offender.getOtherIds().getCrn());
+    }
+
+    // TODO - when offender search gives us the probation status we can remove this method
+    @Deprecated(forRemoval = true)
     private SearchResponse combine(Tuple2<SearchResponse, ProbationStatusDetail> tuple2) {
         SearchResponse searchResponse = tuple2.getT1();
-        return SearchResponse.builder().matchedBy(tuple2.getT1().getMatchedBy())
-            .probationStatusDetail(tuple2.getT2())
-            .matches(searchResponse.getMatches())
-            .build();
+        ProbationStatusDetail probationStatus = tuple2.getT2();
+
+        if (searchResponse.isExactMatch() && !featureFlags.offenderSearchForProbationStatus()) {
```

Comment:
If there is one match and we are not using offender search, we effectively have to get the value from court-case-service but we will map that result from court-case-service into the exact match so that it looks like it came from offender-search. This can all go after we remove the flag.